### PR TITLE
create event with multiple attendees

### PR DESCRIPTION
**Pull request by yogesh-pawar-2023**

### CODE REVIEW

File: src/src/events/events.service.ts [hunk 1]
```diff
@@ -34,78 +33,84 @@ export class EventsService {
     'start_time',
     'updated_by',
     'user_id',
+    "reminders"
+  ];
+
+  public attendancesReturnFields = [
```
Comment (manojLondhe) [inline src/src/events/events.service.ts:39]: rename as `attendanceReturnFields`

File: src/src/events/events.service.ts [hunk 2]
```diff
@@ -34,78 +33,84 @@ export class EventsService {
...
       query.push(obj)
     }
-
-    console.log("query", query)
-
-    const promises = []
-
     for (const iterator of query) {
-      promises.push(this.hasuraService.create(this.table, iterator, this.returnFields))
+      promises.push(this.hasuraService.create('attendance', iterator, this.attendancesReturnFields))
```
Comment (manojLondhe) [inline src/src/events/events.service.ts:90]: rename as `attendanceReturnFields`

File: src/src/events/events.service.ts [hunk 3]
```diff
@@ -34,78 +33,84 @@ export class EventsService {
...
     }
-
     const createEvents = await Promise.all(promises)
```
Comment (manojLondhe) [inline src/src/events/events.service.ts:92]: createEvents => createAttendees

File: src/src/events/events.service.ts [hunk 4]
```diff
@@ -34,78 +33,84 @@ export class EventsService {
...
-    console.log("createEvents", createEvents)
-
-    if (createEvents) {
+  if (createEvents) {
       return response.status(200).send({
         success: true,
-        message: 'Events created successfully!',
-        data: createEvents,
+        message: 'Event created successfully!',
+        data: {events:eventResult.events,attendances:createEvents},
```
Comment (manojLondhe) [inline src/src/events/events.service.ts:98]: attendances => attendance

Comment (yogesh-pawar-2023) [reply src/src/events/events.service.ts:98]: updated as per comment

File: src/src/events/events.service.ts (hunk 3 above)
Comment (yogesh-pawar-2023) [reply src/src/events/events.service.ts:92]: updated name

File: src/src/events/events.service.ts (hunk 2 above)
Comment (yogesh-pawar-2023) [reply src/src/events/events.service.ts:90]: update name

File: src/src/events/events.service.ts (hunk 1 above)
Comment (yogesh-pawar-2023) [reply src/src/events/events.service.ts:39]: updated array name as per comment

**Pull request merged**